src/test/java/RoundTripSearchTest.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+import pages.MainPage;
+import pages.FlightSearchPage;
+
+import static org.testng.Assert.assertTrue;
+
+public class RoundTripSearchTest {
+    private WebDriver driver;
+    private MainPage mainPage;
+    private FlightSearchPage flightSearchPage;
+
+    @BeforeClass
+    public void setup() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://uzairways.com/en");
```

Comment:
Url is common for the whole application and all pages, right?

URI there might differ but base url is common. I'd recommend to create a constant variable storing base url and create URI inside page classes with uri info. uri is related to page, right?